<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentConfig</class>
 <widget class="QWidget" name="AgentConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>369</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>750</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelAgent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>连接方式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agentDisconnect">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>断开连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelURI">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>127</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>URI</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="agentTestResult">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;error or ok&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agentTest">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>连接测试</string>
   </property>
  </widget>
  <widget class="QComboBox" name="agentResources">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="agentConnect">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>建立连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agentReconnect">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>重新连接</string>
   </property>
  </widget>
  <widget class="QComboBox" name="agentList">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>agent</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="agentUri">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelResource">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>resource</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
